server.ts

```diff
@@ -1,25 +1,25 @@
-import {readFile, writeFile} from "fs"
-import tmi from "tmi.js"
-import express from "express"
-import type { Request, Response } from "express"
-import cors from "cors"
+import { readFile, writeFile } from "fs";
+import tmi from "tmi.js";
+import express from "express";
+import type { Request, Response } from "express";
+import cors from "cors";
 import EventEmitter from "events";
 
-var process = require('process')
-process.on('SIGINT', () => {
-  console.info("Interrupted")
-  process.exit(0)
+var process = require("process");
+process.on("SIGINT", () => {
+  console.info("Interrupted");
+  process.exit(0);
 });
 
 /* Bump this number, it will cause any connected browsers to reload after app restart. */
 const protocol = 5;
 const react = new EventEmitter();
-
+/* solo permite 1 tarea por usuario, no varias */
```

Comment:
It's ok, but if the PR changes how the state works, this comment shouldn't actually be here anymore (because now it permits more than 1)